### PR TITLE
Web3 Bug Fixes

### PR DESCRIPTION
**Pull request by johnhalbert**
Resolves the following issues from #818 : 1, 2, 4, 5, 6, 9

### CODE REVIEW

File: src/components/web3/StartCall.tsx [hunk 1]
```diff
@@ -53,17 +55,24 @@ export const StartCall: React.FC<Props> = ({
 
       web3NFTs(web3Address)
         .then(setNfts)
-        .catch((err) => console.error("!!! failed to fetch NTFs ", err));
+        .catch((err) => {
+          console.error("!!! failed to fetch NTFs ", err);
```
Comment (mrose17) [inline src/components/web3/StartCall.tsx:59]: s/NTFs/NFTs

**Review by mrose17 — CHANGES_REQUESTED** [review]
Please the typo and ensure that the links change when we change the text on non-subscriber/subscriber status...

File: src/components/web3/Web3CTA.tsx [hunk 2]
```diff
@@ -21,7 +22,11 @@ export const Web3CTA: React.FC<Props> = ({ onClick }) => {
           avatar, assign moderator privileges using POAPs, and more.
         </p>
         <Button hollow onClick={onClick}>
-          Start a Web3 Call
+          {isSubscribed ? (
+            <>Start a Web3 Call</>
+          ) : (
+            <>Start a Free Trial To Start a Web3 Call</>
+          )}
```
Comment (tackley) [inline src/components/web3/Web3CTA.tsx:29]: Shouldn't this also cause the button to navigate to the account.brave.com if you're not subscribed? Although the message has changed, it still works.

Comment (tackley) [reply src/components/web3/Web3CTA.tsx:29]: Oh just seen @mrose17 said exactly the same thing, apologies.

**Review by mrose17 — APPROVED** [review]
Comments addressed, thanks!

**Pull request merged**